src/openzaak/components/documenten/admin.py

```diff
@@ -23,6 +26,22 @@
 )
 
 
+class FileDownloadMixin:
```

Comment:
Please don't make _everything_ a mixin. There is only one place where this is used now - this should really just be a method on the `ModelAdmin` where it's used.

Reply:
I figured since I added it to both the `EnkelvoudigInformatieObjectInline` and the `EnkelvoudigInformatieObjectAdmin`

Reply:
my bed, I missed that in the diff then :) 😅